runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_thieving.json

```diff
@@ -210,12 +222,24 @@
       "name": "Gems Stall",
       "xp": 160
     },
+    {
+      "level": 78,
+      "icon": 1623,
+      "name": "Dorgesh-kaan rich chest",
```

Comment:
```suggestion
      "name": "Dorgesh-Kaan Rich chest",
```

Reply:
Might as well capitalize the `c` in `chest` while you're at it to stay consistent with the rest of the file